.vsts-dotnet.yml

```diff
@@ -55,7 +55,7 @@ steps:
   displayName: Run build.cmd
   inputs:
     filename: '$(comspec)'
-    arguments: '/c "call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat" & $(Build.Repository.LocalPath)\build.cmd -skiptests -bootstraponly -pack -sign -configuration Release -properties /p:SignType=$(SignType)"'
+    arguments: '/c "call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat" & $(Build.Repository.LocalPath)\build.cmd -pack -sign -configuration Release -properties /p:SignType=$(SignType)"'
```

Comment:
I think `VS140COMNTOOLS` might be a usable variable instead of  `C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools`

Reply:
`VS140COMNTOOLS` is dev14/VS 2015, so it's not a good option for us (we need the 2017/dev15 environment). The _right_ thing to do here is use `vswhere.exe` to find VS and call the script from there. We should do that but it'll keep for now, I think.